src/fsharp/FSComp.txt

```diff
@@ -572,7 +572,7 @@ tcUseWhenPatternGuard,"Character range matches have been removed in F#. Consider
 736,tcExprUndelayed,"TcExprUndelayed: delayed"
 737,tcExpressionRequiresSequence,"This expression form may only be used in sequence and computation expressions"
 738,tcInvalidObjectExpressionSyntaxForm,"Invalid object expression. Objects without overrides or interfaces should use the expression form 'new Type(args)' without braces."
-739,tcInvalidObjectSequenceOrRecordExpression,"Invalid object, sequence or record expression"
+#739,tcInvalidObjectSequenceOrRecordExpression,"Invalid object, sequence or record expression"
```

Comment:
What is the significance of the `#`?

Reply:
It's commented out.

Reply:
This should not be commented out. We should either remove it or keep it.